src/docs/arch-rts.dox

```diff
@@ -0,0 +1,112 @@
+/*! @arch_page arch-rts Rollback to stable
+
+Rollback to stable is an operation that retains only the modifications that
+are stable according to the stable timestamp and recovered checkpoint snapshot.
```

Comment:
Can we add a note saying - Snapshot details are saved at the end of every checkpoint and that is referred to as recovered checkpoint snapshot.?

Reply:
Added.